src/btf.cpp

```diff
@@ -181,21 +182,21 @@ void BTF::load_module_btfs(const std::set<std::string> &modules)
 
 static void dump_printf(void *ctx, const char *fmt, va_list args)
 {
-  auto *ret = static_cast<std::string *>(ctx);
+  auto *ret = static_cast<std::stringstream *>(ctx);
```

Comment:
nit: These minor stream changes feel like they belong in their own commit.

Reply:
I'd ordinarily split this out, I just have too much building up here. The commit is pretty trivial (and this is needed to make it not dog-slow when generating `vmlinux.h`), so I'm going to move ahead. There's going to be a lot of churn in BTF anyways.